packages/arb-token-bridge-ui/src/hooks/CCTP/useUpdateUsdcBalances.ts

```diff
@@ -0,0 +1,135 @@
+import { useCallback, useMemo } from 'react'
+import { isAddress } from 'ethers/lib/utils.js'
+import { Address } from 'wagmi'
+import useSWRImmutable from 'swr/immutable'
+
+import { CommonAddress } from '../../util/CommonAddressUtils'
+import { getL2ERC20Address } from '../../util/TokenUtils'
+import { useNetworks } from '../useNetworks'
+import { useNetworksRelationship } from '../useNetworksRelationship'
+import { isNetwork } from '../../util/networks'
+import { useBalances } from '../useBalances'
+import { getProviderForChainId } from '@/token-bridge-sdk/utils'
+
+export async function childChainUsdcAddressFetcher([
+  _parentChainUsdcAddress,
```

Comment:
I wonder if we can do something about `_parentChainUsdcAddress`, it should be required for non-Orbit chains, so we should be able to just call the method without specifying parent USDC address and rely on chain ID only.

Maybe we don't need `useParentChainUsdcAddress`, we could just make it a util method and use it here as well?